examples/IBFE/explicit/ex5/example.cpp

```diff
@@ -693,58 +726,87 @@ postprocess_data(Pointer<Database> input_db,
         }
         get_values_for_interpolation(x_node, *x_ghost_vec, dof_indices);
         get_values_for_interpolation(U_node, *U_ghost_vec, dof_indices);
+        get_values_for_interpolation(X0_node, X0_vec, dof_indices);
+        if (use_pressure_jump_conditions && use_velocity_jump_conditions)
+            get_values_for_interpolation(TAU_node, *TAU_ghost_vec, dof_indices);
 
         const unsigned int n_qp = qrule->n_points();
         for (unsigned int qp = 0; qp < n_qp; ++qp)
         {
+            interpolate(X, qp, X0_node, phi);
             interpolate(x, qp, x_node, phi);
             jacobian(FF, qp, x_node, dphi);
             interpolate(U, qp, U_node, phi);
+            if (use_pressure_jump_conditions && use_velocity_jump_conditions) interpolate(TAU, qp, TAU_node, phi);
             for (unsigned int d = 0; d < NDIM; ++d)
             {
                 U_qp_vec[d] = U(d);
             }
-            tether_force_function(F, FF, x, q_point[qp], elem, var_data, grad_var_data, loop_time, tether_data_ptr);
+            if (use_boundary_mesh)
+                tether_force_function(F, n, N, FF, x, X, elem, 0, var_data, grad_var_data, loop_time, tether_data_ptr);
+            else
+                tether_force_function(F, n, x, q_point[qp], elem, var_data, grad_var_data, loop_time, tether_data_ptr);
+
             for (int d = 0; d < NDIM; ++d)
             {
                 F_integral[d] += F(d) * JxW[qp];
+                if (use_pressure_jump_conditions && use_velocity_jump_conditions) T_integral[d] += TAU(d) * JxW[qp];
             }
         }
-        for (unsigned short int side = 0; side < elem->n_sides(); ++side)
+        if (!use_boundary_mesh)
         {
-            if (elem->neighbor_ptr(side)) continue;
-            fe_face->reinit(elem, side);
-            const unsigned int n_qp_face = qrule_face->n_points();
-            for (unsigned int qp = 0; qp < n_qp_face; ++qp)
+            for (unsigned short int side = 0; side < elem->n_sides(); ++side)
```

Comment:
This is not the correct algorithm for evaluating lift/drag forces for a volumetric mesh. Please revert these changes.